common/lib/config_definition.py

```diff
@@ -461,7 +461,13 @@
         "default": True,
         "tooltip": "If a dataset is a JSON file but it can be mapped to a CSV file, show the CSV in the preview instead"
                    "of the underlying JSON."
-    }
+    },
+    "ui.option_email": {
+        "type": UserInput.OPTION_TOGGLE,
```

Comment:
Perhaps make this an OPTION_CHOICE with choices 'no e-mails', 'datasets only' and 'datasets and processors'?

Reply:
That's a good idea. I'll do that.